libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/SkillMocks/SkillClaimMock.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Connector.Authentication;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.SkillMocks
+{
+    public class SkillClaimMock : SkillMock
```

Comment:
Needs Code Comments on all public classes and methods.

Reply:
removed